specification/timeseriesinsights/resource-manager/readme.md

```diff
@@ -124,6 +137,7 @@ batch:
   - tag: package-2017-11-15
   - tag: package-2017-02-preview
   - tag: package-2018-08-preview
+  - tag: package-2020-05-15
```

Comment:
Please also add a `### Tag: package-2020-05-15 and java` below.

Reply:
Thanks! Just did it.